sentry_sdk/integrations/starlite.py

```diff
@@ -15,6 +14,7 @@
     from starlite.plugins.base import get_plugin_for_value  # type: ignore
     from starlite.routes.http import HTTPRoute  # type: ignore
     from starlite.utils import ConnectionDataExtractor, is_async_callable, Ref  # type: ignore
+    from pydantic import BaseModel  # type: ignore
```

Comment:
Does `starlite` depend on `pydantic`? Trying to understand why we need to move this line into the `try` block

Reply:
Yes, it uses `pydantic` for valdidation and parsing.